src/app/components/search/search.rs

```diff
@@ -190,6 +190,7 @@ impl SearchResults {
                 self.album_results_model.append(&AlbumModel::new(
                     &album.artists_name(),
                     &album.title,
+                    album.year(),
```

Comment:
My Rust is rusty, but isn't there supposed to be an ampersand here? I think this moves album.year as opposed to referencing it.

Reply:
`album.year()` returns an `u32` and yes, it's moved into the `AlbumModel`. `u32` is always small while strings can be sometimes larger so they are passed over references I suppose. 

`SongModel::new` also takes `u32` and its value is given without an ampersand. In this case `track_number` is implicitly copied into `SongModel` since `u32` implements `Copy`.
https://github.com/xou816/spot/blob/3fa78e951881de05a19940fe6a916a3b51a0ceda/src/app/models/main.rs#L41